Automattic/jetpack
# VideoPress: first pagination approach for Local videos

This PR implements the first pagination approach for the Local videos.

Fixes #

#### Changes proposed in this Pull Request:

* This PR implements the first pagination approach for the Local videos:
  * Add selectors, resolvers, actions and reducers
  * propagate the data, and expose them though the hook
  * print fresh data after user interaction

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the VideoPress library
* Ensure you have local videos
* Ensure you have more than six videos :-D
* Confirm the Pagination is according to the total of videos
* Confirm the list gets refreshed once by changing the page
* Take a look at the network and Redux dev tool


https://user-images.githubusercontent.com/77539/195405117-7a7d7165-b8c9-4ee6-8398-6c358a212c16.mov
